### PR TITLE
Disallow usages of old FIDL syntax

### PR DESCRIPTION
**Pull request by felixzhuologist**
As the penultimate step of the FIDL syntax migration,
block new usages of the old syntax. This will allow
us to remove support for the old syntax and
remove the experimental flag as the final cleanup
step.

### CODE REVIEW

**Review by chinmaygarde — APPROVED** [review]

**Comment by felixzhuologist** [comment]
Thanks for taking a look - will let you know when this is ready to land!

**Comment by chinmaygarde** [comment]
Sounds good. Is there anything specific this is blocked on?

**Comment by felixzhuologist** [comment]
This PR should be safe to merge now, would you be able to merge this for me?
Wanted to ensure that no old syntax files have snuck in before landing this, but things should be good to go now :)

**Pull request merged**
